components/Wrapper.js

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+import ArtGenerator from "./ArtGenerator";
+
+export default function Wrapper() {
+  return (
+    <StyledDiv>
+      <ArtGenerator />
+    </StyledDiv>
+  );
+}
+
+const StyledDiv = styled.div`
+  color: green;
+  background-color: white;
+  width: 376px;
```

Comment:
```suggestion
  width: 375px;
```
If that unit is about the recommended width for the mobile view, you're one pixel off. ;)
For now and for the first couple of capstone weeks this is all fair and good, but perhaps try not to hardcode device screen widths into your CSS. 👀 💦